frontend/packages/console-shared/src/components/pod/PodStatus.tsx

```diff
@@ -97,6 +97,7 @@ const PodStatus: React.FC<PodStatusProps> = ({
           duration: ANIMATION_DURATION,
           onEnd: updateOnEnd ? forceUpdate : undefined,
         }}
+        ariaTitle={`${truncTitle} ${truncSubTitle}`} // ex: '3 pods' or '3 Scaling to 2'
```

Comment:
Should we use the untruncated value for screen readers? Presumably it's truncated because it won't fit inside the donut. Here we don't have that problem.

Reply:
fixed, thanks